README.md

```diff
@@ -11,19 +11,36 @@ and attacher for Ceph RBD and CephFS.
 
 ## Overview
 
-Ceph CSI plugins implement an interface between CSI enabled Container
-Orchestrator (CO) and CEPH cluster.
-It allows dynamically provisioning CEPH volumes and attaching them to
-workloads.
-Current implementation of Ceph CSI plugins was tested in Kubernetes
-environment (requires Kubernetes 1.13+), but the code does not rely on
-any Kubernetes specific calls (WIP to make it k8s agnostic) and
-should be able to run with any CSI enabled CO.
+Ceph CSI plugins implement an interface between CSI enabled Container Orchestrator
+(CO) and CEPH cluster. It allows dynamically provisioning CEPH volumes and
+attaching them to workloads.
```

Comment:
@ShyamsundarR  CEPH volume are ambiguous, we may have to mention about CephFS and RBD to be precise.

Reply:
The next line clarifies the same.